config/jobs/kubernetes/wg-k8s-infra/canaries/sig-node-containerd.yaml

```diff
@@ -59,6 +59,33 @@ periodics:
     testgrid-dashboards: wg-k8s-infra-canaries
     testgrid-tab-name: containerd-build-1.4-canary
     description: "builds release/1.4 branch of upstream containerd"
+- name: ci-containerd-build-1-5-canary
+  interval: 30m
+  cluster: k8s-infra-prow-build
+  labels:
+    preset-service-account: "true"
```

Comment:
I think we need to add `preset-dind-enabled: "true"` as well 
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/wg-k8s-infra/canaries/sig-node-containerd.yaml#L2-L32

Reply:
Done.